test/unit/specs/components/common/PagePreferences.spec.js

```diff
@@ -66,20 +66,50 @@ describe(`PagePreferences`, () => {
     expect(wrapper.find(`#onboarding`)).toBeDefined()
   })
 
-  it(`switches mocked mode`, () => {
-    wrapper.vm.networkSelectActive = `mock`
-    wrapper.vm.setMockedConnector()
-    expect(wrapper.vm.mockedConnector).toBe(true)
-    expect(store.dispatch).toHaveBeenCalledWith(`setMockedConnector`, true)
+  describe(`switches mocked mode`, () => {
+    it(`mock`, () => {
```

Comment:
I think it would be beneficial to make these test names a little more speaking. What do you think?

Reply:
Done.